### PR TITLE
Merge 2.5.0 SDK to github-maven-repository

### PR DESCRIPTION
**Pull request by williamexperian**

### CODE REVIEW

File: maven/com/experian/datastudio/sdklib/2.5.0/sdklib-2.5.0.pom [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.experian.datastudio</groupId>
+    <artifactId>sdklib</artifactId>
+    <version>2.5.0</version>
+    <packaging>jar</packaging>
+    <name>sdklib</name>
+    <url>https://www.edq.com/documentation/aperture-data-studio/</url>
+    <dependencies>
+		<dependency>
```
Comment (chungkhenhah) [inline maven/com/experian/datastudio/sdklib/2.5.0/sdklib-2.5.0.pom:12]: alignment

Comment (williamexperian) [reply maven/com/experian/datastudio/sdklib/2.5.0/sdklib-2.5.0.pom]: Thank you @chungkhenhah, I have fixed the indentation :)

**Comment by izzulhaziq** [comment]
LGTM

**Review by izzulhaziq — APPROVED** [review]
LGTM

**Pull request merged**